erts/preloaded/src/erts_literal_area_collector.erl

```diff
@@ -81,12 +89,13 @@ msg_loop(Area, Outstnd, GcOutstnd, NeedGC) ->
 	%% Unexpected garbage message. Get rid of it...
 	_Ignore ->
 	    msg_loop(Area, Outstnd, GcOutstnd, NeedGC)
-
+    after HibernateTmo ->
+            %% We hibernate in order to clear the heap completely.
+            erlang:hibernate(?MODULE, start, [])
```

Comment:
This one is problematic. We cannot hibernate unless `Outsntd == 0`, i.e. we have no outstanding replies. If we hibernate with outstanding replies we might release the literal area we are working on too early.

Reply:
It will never do that as then `HibernateTmo` will be `infinity`.

Reply:
Ahh missed that